### ISSUE TITLE
Where can I get example datasets to train Rasa - NLU?

### ISSUE THREAD

**Issue by shgidi** [RasaHQ/rasa_nlu#561]
**rasa NLU version** (e.g. `0.7.3`):

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...):

**Operating system** (windows, osx, ...):

**Issue**:

**Content of configuration file** (if used & relevant):
```json

```

**Comment by wrathagom** [comment]
There are a few examples in the data folder in the repo. Other than that I am not aware of any that are pre-made.

You could probably use API.ai's small talk domain that they opened: https://chatbotsmagazine.com/api-ai-small-talk-is-now-open-source-why-is-it-a-big-deal-a39c54f89d85

Any reason you don't just want to Create your own, the docs on the formatting are helpful: https://rasa-nlu.readthedocs.io/en/latest/dataformat.html 

You can use the online "trainer" to create the training data in the correct format: https://rasahq.github.io/rasa-nlu-trainer/

Especially with the MITIE backend not many examples per intent are required.

**Comment by shgidi** [comment]
Thanks for the quick answer.
I currently examining the library, and before I start making my own data-sets (my main purpose is using it on Hebrew :) ) I want to try it on some pre-maid data-sets, to get a feel of the results.
I think public available data-sets (on top of the single set in the repo) will be a great boost to the library.

**Comment by amn41** [comment]
also snips have a pretty cool open dataset : https://github.com/snipsco/nlu-benchmark

**Comment by projectlab1** [comment]
@amn41 snips dataset format is not consistent with rasa_nlu ...... Using it would require manually changing the dataset ?

**Comment by wrathagom** [comment]
yes you would have to modify it, no there is not a large dataset publicly available for Rasa that we know of. If you take the time to modify the snips dataset then you may be able to contribute that back to this repo as a good test set for future users.

**Comment by pandeysudhanshu** [comment]
I always get a syntax error when i try to create a dataset

**Comment by tmbo** [comment]
Please create a separate issue with details about what you are trying to do and the error you are getting.
